code/controllers/verbs.dm

```diff
@@ -42,7 +42,7 @@
 			debug_variables(evacuation_controller)
 			feedback_add_details("admin_verb","DEmergency")
 		if("Configuration")
-			debug_variables(config)
+			debug_variables(global.config)
```

Comment:
why is this global.config instead of GLOB.config?

Reply:
> why is this global.config instead of GLOB.config?

Because GLOBAL_REAL(config, /configuration) <=> var/global/configuration/config